addons/default/visiosoft/advs-module/resources/css/list.css

```diff
@@ -212,4 +212,8 @@ a.sort-by-open-dropdown:hover {
 .sort-by-item:hover {
     text-decoration: underline;
     cursor: pointer;
+}
+
+.filter-section .countries input[type=checkbox] {
+    display: none;
 }
```

Comment:
don't hide it, remove it, you can select one country why use checkbox

Reply:
it's not logical right but next week will be multiple selectcable like alibaba it cloud stay like that @diashalabi